schema/bundle.schema.json

```diff
@@ -187,6 +187,10 @@
             "description": "A description of the purpose of this action",
             "type": "string"
           },
+          "display": {
```

Comment:
This should change to `title`, correct?

Reply:
Ugh... I missed a file in my commit. Sorry. Will fix that.